esrc/search.php

```diff
@@ -130,9 +130,10 @@
 <div class="container">
 
 <div class="row" id="header" style="padding-top: 10px;">
+	<?php include_once '../includes/top-right.php'; ?>
```

Comment:
I think the order is important (not sure). But this includes the php to the location.

Reply:
Order does matter.  the float right needs to go first.